resources/eventing/profile-production.yaml

```diff
@@ -13,19 +13,20 @@ controller:
     maxMessages: -1
   resources:
     limits:
-      cpu: 1000m
-      memory: 512Mi
+      cpu: 2000m
+      memory: 1Gi
     requests:
-      cpu: 10m
+      cpu: 50m
       memory: 256Mi
   publisherProxy:
+    replicas: 2
     resources:
       limits:
         cpu: 500m
         memory: 512Mi
       requests:
-        cpu: 10m
-        memory: 256Mi
```

Comment:
the request should not be higher than the limits. why are the cpu numbers so high?

Reply:
My bad, I supposed to raise the limit & instead I increased request. I have corrected. Now cpu request is moved backed to 50m & limit is 1000m, we are keeping it high to allow bigger bursts.